packages/sdk-core/src/modules/data/BacktraceDataBuilder.ts

```diff
@@ -21,6 +22,13 @@ export class BacktraceDataBuilder {
     ): BacktraceData {
         const reportData = ReportDataBuilder.build(report.attributes);
 
+        const stackTrace = this._stackTraceConverter.convert(report.stackTrace, report.message);
+
+        this._debugIdProvider.loadDebugIds();
+        for (const frame of stackTrace) {
+            frame.debug_identifier = this._debugIdProvider.getDebugId(frame.library);
+        }
+
         return {
```

Comment:
important note before we land: 

debug_identifier with our model always goes with the modules declaration in global annotations. The modules declaration allows us to match specific debug_id to the library stored in our symbols system. I know, it might work differently for source maps, however, for every other integration one piece of information requires the other.

Reply:
I talked with George and he required that frames only have debug_identifier fields.